### PR TITLE
[SDMEXT-801] Update Readme

### PR DESCRIPTION
**Pull request by rishikunnath2747**
This PR adds a note mentioning how to change the cds-services-version in the application pom, with a reference to the @cap-java/sdm deploy branch pom.xml

Link to JIRA BL : [SDMEXT-801](https://jira.tools.sap/browse/SDMEXT-801)

### CODE REVIEW

**Review by yashmeet29 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 * [MTAR builder](https://www.npmjs.com/package/mbt) (`npm install -g mbt`)
 * [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html), Install cf-cli and run command `cf install-plugin multiapps`.
 
+> [!Note]
+> Reading attachment behaviour depends on the version of cds-services version which is used by the CAP application.
```
Comment (dheerasameerk) [inline README.md]: You can rephrase this to:
The behaviour of clicking attachment and opening it varies based on the version of cds-services used by the CAP application.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,11 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
...
+For versions < 3.4.0, clicking on attachment will download the file to the computer.
+For versions >= 3.4.0, clicking on attachment will open the file in new tab on browser, if browser supports the file type. A reference to adding this can be found in the [deploy branch](https://github.com/cap-java/sdm/blob/691c329f4c3c17ae390cfcb2db1ef02650585aee/cap-notebook/demoapp/pom.xml#L20)
```
Comment (dheerasameerk) [inline README.md]: Interchange the points, let >=3.4.0 be the first item.
Rephrase to ---> 
For cds-services version >= 3.4.0, clicking on attachment will 
- open the file in new browser tab, if browser supports the file type.
- download the file to the computer, if browser does not support the file type.

For cds-services version < 3.4.0, clicking on attachment will download the file to the computer

Also the text for the pom.xml reference can be changed to "here" instead of "deploy branch".

File: README.md (hunk 1 above)
Comment (rishikunnath2747) [reply README.md]: Changed

File: README.md [hunk 3]
```diff
@@ -29,6 +29,15 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 * [MTAR builder](https://www.npmjs.com/package/mbt) (`npm install -g mbt`)
 * [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html), Install cf-cli and run command `cf install-plugin multiapps`.
 
+> [!Note]
+> The behaviour of clicking attachment and opening it varies based on the version of cds-services used by the CAP application. For cds-services version >= 3.4.0, clicking on attachment will
```
Comment (dheerasameerk) [inline README.md]: Add the second line as bullet point. For example,

- For version >= 3.4.0
- For version < 3.4.0

Also after note, add a heading "cds-services"

**Review by dheerasameerk — APPROVED** [review]
